webhook/command_response.go

```diff
@@ -0,0 +1,60 @@
+package webhook
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"net/http"
+
+	"github.com/micromdm/micromdm/connect"
+	"github.com/micromdm/micromdm/pubsub"
+	"github.com/pkg/errors"
+)
+
+const contentType = "application/x-apple-aspen-mdm"
+
+type commandWebhook struct {
+	Topic string
+	CallbackUrl string
+}
+
+func NewCommandWebhook(topic, callbackURL string) (ListeningWebhook, error) {
+	if topic == "" {
+		return nil, errors.New("topic should not be empty")
```

Comment:
change the new errors to say `webhook: <error message>`
for example: `"webhook: topic should not be empty"`

otherwise it becomes hard to track where the error is coming from.